<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maze Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnminimumwidth="620,150">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="height_lbl">
         <property name="text">
          <string>Maze Height:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="width_entry">
         <property name="text">
          <string>21</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="width_lbl">
         <property name="text">
          <string>Maze Width:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="height_entry">
         <property name="text">
          <string>25</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="solver_lbl">
         <property name="text">
          <string>Solver:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="solver_combo">
         <property name="currentText">
          <string>Backtracking</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Backtracking</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DFS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A* search</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="generateMaze_btn">
       <property name="text">
        <string>Generate
Maze</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QLabel" name="mazeView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="solve_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Solve
Maze</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
